### ISSUE TITLE
Connection state menu item missing on macOS 10.14 and earlier, starting with 1.34.0 release

### ISSUE THREAD

**Issue by mjsprojects** [tailscale/tailscale#6707]
### What is the issue?

I updated to the latest version of the Tailscale Mac client v1.34.0, and it looks like the UI to connect / disconnect has changed. This was previously a button which was easily accessible with the VoiceOver screen reader, but this UI element has changed to a non standard toggle element that the screen reader does not identify and I have no way to read it's status, or toggle it on / off with the keyboard.

### Steps to reproduce

You can enable VoiceOver on the mac with CMD+F5, then press control+option+m twice to bring up the menu and control+option+space to open Tailscale. In the old client you could ontrol+option+down-arrow to the connect button and control+option+space again to activate it, the new UI toggle element is not accessible to keyboard navigation and VoiceOver doesn't even find the element to report its status. CMD+f5 again to disable VoiceOver.

### Are there any recent changes that introduced the issue?

Upgrade from V1.32.2 to V1.34.0 of the Mac client.

### OS

macOS

### OS version

Ventura

### Tailscale version

1.34.0

**Comment by DentonGentry** [comment]
While we work on addressing this, there are two ways to get back to an earlier version before that disconnect control went in:

- to stay on the App Store version, join TestFlight at https://testflight.apple.com/join/7ExrFq00 which will let you go back to any currently active test release. 1.33.289 should work for this.
- if you switch from the App Store to the downloadable app, you can download 1.32.3 from https://pkgs.tailscale.com/stable/Tailscale-1.32.3-macos.zip

**Comment by mjsprojects** [comment]
I have reverted to 1.32.3, thanks for the download link.

**Comment by mihaip** [comment]
Apologies for the disruption. We are using a custom view inside a menu item, which is one of the less documented parts of AppKit. We attempted to make it as similar to a normal menu item as possible, it's navigable with the arrow keys, when focused it should announce "Tailscale, Connected, interactive" (or "Tailscale, Disconnect, interactive" if not currently connected) and it responds to activation (via the space or return keys) to toggle state. 

I tested the above by using control-F8 to move focus to the status menus and then using the arrow keys and space to select and open the Tailscale menu, I will investigate what is different with the activation mechanism that you described.

**Comment by fanuch** [comment]
I've got some clarifications to make.

This issue (if I understand correctly) is less about accessibility in this case, and more about the literal toggle option is missing.

Please see my Tailscale App from the AppStore 1.34.0

![Tailscale2](https://user-images.githubusercontent.com/4865267/207231565-33d0bf13-3ca8-4bdb-8c09-3e6a7fe711b4.png)

Please see @mjsprojects Tailscale App from the AppStore 1.34.0
You can see the the white box from MacOS VoiceOver over the first element which is the "dimmed" key expiry
![Tailscale1](https://user-images.githubusercontent.com/4865267/207231613-bd16c232-ffdc-4822-b11b-a107f8ebef3d.png)


@mjsprojects literally is missing the row that says "Tailscale" and the toggle option.

This may have incorrectly been tagged "accessibility" because from their perspective, the element was not reachable but in actuality the option is missing entirely!

Hope this helps.

**Comment by mjsprojects** [comment]
I have just tested this on an M1 MacBook running the latest version of Ventura and the toggle is accessible with VoiceOver, on an older intel MacBook Air running an older build of Mac OS it is not.

Additionally installing the Tailscale CLI and running 'Tailscale up' is a reasonable workaround for older machines that aren't patched to the latest version of Ventura.

**Comment by mihaip** [comment]
I'm able to reproduce this in a VM running 10.14.6, so I do suspect some kind of AppKit compatibility issue.

**Comment by mihaip** [comment]
The committed fix is to switch back to plain menu items on 10.14 and 10.13. It will be picked up in the next stable release (1.36), and possibly in a 1.34.2 patch release if we end up doing one.

**Comment by mjsprojects** [comment]
Thanks for the update, I appreciate the fast resolution on this one.
